manifest.php

```diff
@@ -30,12 +30,13 @@
     'label' => 'extension-tao-dac-simple',
     'description' => 'extension that allows admin to give access to some resources to other people',
     'license' => 'GPL-2.0',
-    'version' => '6.5.1',
+    'version' => '6.6.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => [
        'taoBackOffice' => '>=3.0.0',
        'generis' => '>=12.15.0',
-       'tao' => '>=40.9.0'
+       'tao' => '>=40.9.0',
+       'taoTaskQueue' => '>=5.1.0',
```

Comment:
Isn't it a breaking change now?
I mean, there could potentially be some configurations, using taoDacSimple, but not the queue.